test/integ/quote-classic.test.ts

```diff
@@ -807,6 +808,67 @@ describe('quote', function () {
             }
           });
 
+          const sendPortionEnabledValues = [true, undefined];
```

Comment:
nit: any reason why we don't do [true, false] here? I guess undefined is falsy but seems a bit weird.